website/docs/d/ec2_managed_prefix_list.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "VPC"
+layout: "aws"
+page_title: "AWS: aws_ec2_managed_prefix_list"
+description: |-
+    Provides details about a specific managed prefix list
+---
+
+# Data Source: aws_ec2_managed_prefix_list
+
+`aws_ec2_managed_prefix_list` provides details about a specific AWS prefix list or
+customer-managed prefix list in the current region.
+
+## Example Usage
+
+### Find the regional DynamoDB prefix list
+
+```hcl
+data "aws_region" "current" {}
+
+data "aws_ec2_managed_prefix_list" "example" {
+  name = "com.amazonaws.${data.aws_region.current.name}.dynamodb"
+}
+```
+
+### Find a managed prefix list using filters
+
+```hcl
+resource "aws_ec2_managed_prefix_list" "example" {
+  name           = "example"
+  max_entries    = 5
+  address_family = "IPv4"
+  
+  entry {
+    cidr_block = "1.0.0.0/8"
+  }
+}
+
+data "aws_ec2_managed_prefix_list" "example" {
+  filter {
+    name   = "prefix-list-name"
+    values = [aws_ec2_managed_prefix_list.example.name]
+  }
+}
```

Comment:
Resource does not exist yet and we prefer not showing the anti-pattern of having a data source reference a managed resource:

```suggestion
data "aws_ec2_managed_prefix_list" "example" {
  filter {
    name   = "prefix-list-name"
    values = ["example"]
  }
}
```